.travis.yml

```diff
@@ -32,10 +32,8 @@ jobs:
         set -eu -o pipefail
         for dir in containers/*/
           do pushd $dir
-          echo "--- Build container $dir"
-          time make container DOCKER_ARGS=--no-cache
           echo "--- Test container $dir"
-          time make test
+          time make test DOCKER_ARGS=--no-cache
```

Comment:
We don't need --no-cache here I don't think, since these aren't actually used in production right?